guides/doc-Upgrading_and_Updating/topics/updating_satellite_server_to_next_minor_version.adoc

```diff
@@ -70,120 +60,209 @@ If you lose connection to the command shell where the upgrade command is running
+# vi /etc/yum.repos.d/redhat-local.repo
+
+[{RepoRHEL7ServerAnsible}]
+name=Ansible {SatelliteAnsibleVersion} RPMs for Red Hat Enterprise Linux 7 Server x86_64
+baseurl=file:///root/satellite_repos/rhel-7-server-ansible-2.9-rpms
+>>>>>>> 072ea2d6 (In-place upgrade in an airgapped server)
+enabled=1
+
+[{RepoRHEL7Server}]
+name=Red Hat Enterprise Linux 7 Server RPMs x86_64
+baseurl=file:///root/satellite_repos/rhel-7-server-rpms
+enabled=1
+
+[{RepoRHEL7ServerSatelliteServerProductVersion}]
+name={ProjectNameX} for RHEL 7 Server RPMs x86_64
+baseurl=file:///root/satellite_repos/rhel-7-server-satellite-6.10-rpms
+enabled=1
+
+[{RepoRHEL7ServerSatelliteMaintenanceProductVersion}]
+name={ProjectName} Maintenance 6 for RHEL 7 Server RPMs x86_64
+baseurl=file:///root/satellite_repos/rhel-7-server-satellite-maintenance-6-rpms
+enabled=1
+
+[{RepoRHEL7ServerSoftwareCollections}]
+name=Red Hat Software Collections RPMs for Red Hat Enterprise Linux 7 Server x86_64
+baseurl=file:///root/satellite_repos/rhel-server-rhscl-7-rpms
+enabled=1
+----
++
+* In the configuration file, replace the `/root/satellite_repos` with the extracted location (if that is in a different directory.)
+* Check the available versions to confirm the next minor version is listed:
++
+[options="nowrap" subs="attributes"]
 ----
 # {foreman-maintain} upgrade list-versions
 ----
 
-. Use the health check option to determine if the system is ready for the upgrade.
-On the first use of this command, `{foreman-maintain}` prompts you to enter the hammer admin user credentials and saves them in the `/etc/foreman-maintain/foreman-maintain-hammer.yml` file.
+. Use the health check option to determine if the system is ready for upgrade.
+On first use of this command, `{foreman-maintain}` prompts you to enter the hammer admin user credentials and saves them in the `/etc/foreman-maintain/foreman-maintain-hammer.yml` file.
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {foreman-maintain} upgrade check --target-version {ProjectVersion}.__z__
+# {foreman-maintain} upgrade check --target-version {TargetVersion}.__z__
 ----
 +
 Review the results and address any highlighted error conditions before performing the upgrade.
 
-. Because of the lengthy update time, use a utility such as `screen` to suspend and reattach a communication session.
+* Because of the lengthy update time, use a utility such as `screen` to suspend and reattach a communication session.
 You can then check the upgrade progress without staying connected to the command shell continuously.
 For more information about using the screen command, see link:https://access.redhat.com/articles/5247[How do I use the screen command?] article in the _Red{nbsp}Hat Knowledge{nbsp}Base_.
 +
 If you lose connection to the command shell where the upgrade command is running, you can see the logged messages in the `{installer-log-file}` file to check if the process completed successfully.
 
-. Perform the upgrade:
+* Perform the upgrade:
```

Comment:
These completely unnecessary changes are truly painful to review. 
It was not needed at all. 
```suggestion
. Perform the upgrade:
```